dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -128,16 +127,16 @@ internal class DokkaSymbolVisitor(
         moduleFiles: Set<KtFile>
     ): DPackage {
         val dri = getDRIFromPackage(packageSymbol)
-        val scope = packageSymbol.getPackageScope()
-        val callables = scope.callables.toList().filterSymbolsInSourceSet(moduleFiles)
-        val classifiers = scope.classifiers.toList().filterSymbolsInSourceSet(moduleFiles)
+        val scope = packageSymbol.packageScope
+        val callables = scope.callables.filterSymbolsInSourceSet(moduleFiles)
+        val classifiers = scope.classifiers.filterSymbolsInSourceSet(moduleFiles)
 
-        val functions = callables.filterIsInstance<KaFunctionSymbol>().map { visitFunctionSymbol(it, dri) }
-        val properties = callables.filterIsInstance<KaPropertySymbol>().map { visitPropertySymbol(it, dri) }
+        val functions = callables.filterIsInstance<KaNamedFunctionSymbol>().map { visitFunctionSymbol(it, dri) }.toList()
```

Comment:
question: is there any doc regarding the difference between `KaNamedFunctionSymbol` and `KaFunctionSymbol`?

Reply:
https://youtrack.jetbrains.com/issue/KT-68882/Analysis-API-Refactor-KaSymbols
> Rename the existing `KaFunctionSymbol` to KaNamedFunctionSymbol
> Rename `KaFunctionLikeSymbol` to KaFunctionSymbol

I think the hierarchy can answer:
![image](https://github.com/Kotlin/dokka/assets/9636937/7509ece9-b807-48d2-9c85-da2f9262274d)
Or what do you mean?